fdbcli/StatusCommand.actor.cpp

```diff
@@ -0,0 +1,1223 @@
+#include "fdbclient/Knobs.h"
+#include "fdbclient/StatusClient.h"
+
+#include "flow/Arena.h"
+#include "flow/FastRef.h"
+#include "flow/ThreadHelper.actor.h"
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+namespace {
+
+std::string getCoordinatorsInfoString(StatusObjectReader statusObj) {
+	std::string outputString;
+	try {
+		StatusArray coordinatorsArr = statusObj["client.coordinators.coordinators"].get_array();
+		for (StatusObjectReader coor : coordinatorsArr)
+			outputString += format("\n  %s  (%s)",
+			                       coor["address"].get_str().c_str(),
+			                       coor["reachable"].get_bool() ? "reachable" : "unreachable");
+	} catch (std::runtime_error&) {
+		outputString = "\n  Unable to retrieve list of coordination servers";
+	}
+
+	return outputString;
+}
+
+std::string lineWrap(const char* text, int col) {
+	const char* iter = text;
+	const char* start = text;
+	const char* space = nullptr;
+	std::string out = "";
+	do {
+		iter++;
+		if (*iter == '\n' || *iter == ' ' || *iter == '\0')
+			space = iter;
+		if (*iter == '\n' || *iter == '\0' || (iter - start == col)) {
+			if (!space)
+				space = iter;
+			out += format("%.*s\n", (int)(space - start), start);
+			start = space;
+			if (*start == ' ' /* || *start == '\n'*/)
+				start++;
+			space = nullptr;
+		}
+	} while (*iter);
+	return out;
+}
+
+std::pair<int, int> getNumOfNonExcludedProcessAndZones(StatusObjectReader statusObjCluster) {
+	StatusObjectReader processesMap;
+	std::set<std::string> zones;
+	int numOfNonExcludedProcesses = 0;
+	if (statusObjCluster.get("processes", processesMap)) {
+		for (auto proc : processesMap.obj()) {
+			StatusObjectReader process(proc.second);
+			if (process.has("excluded") && process.last().get_bool())
+				continue;
+			numOfNonExcludedProcesses++;
+			std::string zoneId;
+			if (process.get("locality.zoneid", zoneId)) {
+				zones.insert(zoneId);
+			}
+		}
+	}
+	return { numOfNonExcludedProcesses, zones.size() };
+}
+
+int getNumofNonExcludedMachines(StatusObjectReader statusObjCluster) {
+	StatusObjectReader machineMap;
+	int numOfNonExcludedMachines = 0;
+	if (statusObjCluster.get("machines", machineMap)) {
+		for (auto mach : machineMap.obj()) {
+			StatusObjectReader machine(mach.second);
+			if (machine.has("excluded") && !machine.last().get_bool())
+				numOfNonExcludedMachines++;
+		}
+	}
+	return numOfNonExcludedMachines;
+}
+
+std::string getDateInfoString(StatusObjectReader statusObj, std::string key) {
+	time_t curTime;
+	if (!statusObj.has(key)) {
+		return "";
+	}
+	curTime = statusObj.last().get_int64();
+	char buffer[128];
+	struct tm* timeinfo;
+	timeinfo = localtime(&curTime);
+	strftime(buffer, 128, "%m/%d/%y %H:%M:%S", timeinfo);
+	return std::string(buffer);
+}
+
+std::string getProcessAddressByServerID(StatusObjectReader processesMap, std::string serverID) {
+	if (serverID == "")
+		return "unknown";
+
+	for (auto proc : processesMap.obj()) {
```

Comment:
Use `const auto&` for range based for loops by default. I realize this is code that you only moved in this PR - thanks for fixing it anyway